src/createStore.js

```diff
@@ -68,7 +68,7 @@ export default function createStore(reducer, initialState) {
 
       isSubscribed = false
       var index = listeners.indexOf(listener)
-      listeners.splice(index, 1)
+      listeners = [ ...listeners.slice(0, index), ...listeners.slice(index + 1) ]
```

Comment:
Do you mind splitting this to two lines please, and then squashing your commits? Thanks.